api/api/controllers/cluster_controller.py

```diff
@@ -109,8 +110,9 @@ async def get_healthcheck(request, pretty=False, wait_for_complete=False, list_n
                           wait_for_complete=wait_for_complete,
                           logger=logger,
                           local_client_arg='lc',
-                          rbac_permissions=request['token_info']['rbac_policies'],
-                          nodes=nodes
+                          rbac_permissions=get_permissions(connexion.request.headers['Authorization']),
```

Comment:
```suggestion
                          rbac_permissions=request['token_info']['rbac_policies'],
```
Same as above. Change this in every endpoint.